README.org

```diff
@@ -170,6 +170,7 @@ Above, all enjoy using Emacs. The community, that more than anything, makes Emac
    - [[https://github.com/escherdragon/sunrise-commander][Sunrise Commander]] - Twin-pane file manager for Emacs based on Dired and inspired by Midnight Commander.
    - [[https://github.com/Alexander-Miller/treemacs][Treemacs]] - a tree layout file explorer for Emacs.
    - [[https://github.com/sebastiencs/sidebar.el][Sidebar.el]] - A customizable file explorer with git integration for emacs.
+   - [[https://github.com/raghavgautam/tramp-hdfs][tramp-hdfs]] - Browser HDFS in Emacs with dired using Tramp 
```

Comment:
@tninja make sure that lines end with a period: "dired using Tramp."

Also, the readme of @raghavgautam's project states "Browse HDFS(Hadoop Distributed File System) in Emacs using Tramp." That being said "Browser HDFS in Emacs using Tramp" sounds better as "Browse HDFS in Emacs using Tramp."